client/lib/plans/features-list.js

```diff
@@ -572,10 +572,7 @@ export const FEATURES_LIST = {
 				},
 			} ),
 		getDescription: () =>
-			i18n.translate(
-				"With increased storage space you'll be able to upload " +
-					'more images, audio, and documents to your website.'
-			),
+			i18n.translate( 'Upload more images, audio, and documents to your website.' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **2** times:
`translate( 'Upload more images, audio, and documents to your website with increased storage space.' )` *ES Score: 6*